test/missing_flag_name.cc

```diff
@@ -23,3 +23,25 @@ TEST(FlagsTest, MissingFlagHelp) {
       "Missing flag 'help' for "
       "field 'test.MissingFlagHelp.s'");
 }
+
+TEST(FlagsTest, MissingSubcommandFlagName) {
+  EXPECT_DEATH(
+      []() {
+        test::SubcommandFlagsWithoutName missing_flag_name;
+        auto builder = stout::flags::Parser::Builder(&missing_flag_name);
+        builder.Build();
+      }(),
+      "Missing at least one flag name in 'names' for "
```

Comment:
```suggestion
      "Missing at least one subcommand name in 'names' for "
```

Reply:
Check `void Parser::TryFillFieldAndMessageHelpers()` in [this](https://github.com/3rdparty/stout-flags/pull/18) PR.